pkg/scripts/generate-variables.sh

```diff
@@ -27,7 +27,7 @@ set_variables() {
   AWS_ACCOUNT_ID=$(aws sts get-caller-identity --query Account --output text)
 
   # Generates the unique s3 bucket and key names for the deployment for keeping the terraform state
-  S3_BUCKET="atlas-${ENVIRONMENT_NAME}-${REGION}-${AWS_ACCOUNT_ID}-tfstate"
+  S3_BUCKET="atlas-${ENVIRONMENT_NAME}-${REGION}-${AWS_ACCOUNT_ID}-tfst"
```

Comment:
```suggestion
  S3_BUCKET="atlas-tf-${ENVIRONMENT_NAME}-${REGION}-${AWS_ACCOUNT_ID}"
```

Reply:
To be consistent to other naming conventions I leave it as postfix